portfolio/src/main/webapp/pictures.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div id="content">
+        <h1>Catalina's Gallery</h1>
+        <div class="together">
+            <a href="images/luna1.jpg"><img src="images/luna1.jpg"/></a>
+            <a href="images/park1.jpg"><img src="images/park1.jpg"/></a>
+            <a href="images/park2.jpg"><img src="images/park2.jpg"/></a>
+            <a href="images/park3.jpg"><img src="images/park3.jpg"/></a>
+            <a href="images/chewy1.jpg"><img src="images/chewy1.jpg"/></a>
+            <a href="images/park4.jpg"><img src="images/park4.jpg"/></a>
+            <a href="images/park6.jpg"><img src="images/park6.jpg"/></a>
+            <a href="images/roanoke.jpg"><img src="images/roanoke.jpg"/></a>
+            <a href="images/luna2.jpg"><img src="images/luna2.jpg"/></a>
+            <a href="images/park5.jpg"><img src="images/park5.jpg"/></a>
```

Comment:
It might be nice to add "alt" tags here for each image, just in case users have slow connections or other issues with images loading. This is also important for accessibility because visually impaired users won't know what the image is showing.